docs/pages/blog/mui-product-comparison.md

```diff
@@ -0,0 +1,164 @@
+
+We're also in the early stages of developing a low-code internal tool builder called [MUI Toolpad](https://mui.com/toolpad/), in which you can build with every Core and X component in a drag-and-drop interface.
+
+Read on for more details on each of our products.
+
+## MUI Core
+
+The Core is MUI's foundational product line.
+It grew out of Material UI, and that library's legacy lives on in the name of the repo on GitHub: [`mui/material-ui`](https://github.com/mui/material-ui).
+
+But this repo contains much more than just Material UI these days.
+More recent additions include Joy UI and MUI Base, as well as our in-house styling solution, MUI System.
+
+MUI Core is open source, and we invite you to contribute wherever you see fit!
+
+### Material UI
+
+Material UI is an open-source React component library that implements Google's Material Design.
+It includes a comprehensive collection of prebuilt components that are ready for use in production right out of the box.
+
+Material UI is beautiful by design and features a suite of customization options that make it easy to implement your own custom design system on top of our components.
+
+Get started in the [Material UI docs](/material-ui/getting-started/overview/).
+
+#### Key features
+
+- **Material Design:** Your app will look and feel excellent by default, thanks to our meticulous implementation of Material Design.
+- **Comprehensiveness:** with over 50 foundational components and counting, you've got everything you need to ship new features fast.
+- **Maturity:** Material UI's age and maturity rival that of React itself, with its origins spanning all the way back to 2014.
+- **Community:** Over 2,500 open-source contributors have made this library what it is today.
+
+#### Ideal use cases
+
+- User interfaces that adhere closely to Material Design
+- Internal/admin tools
+- Dev teams that need to ship features in hours rather than weeks
+
+### Joy UI
+
+Joy UI is an open-source React component library that implements MUI's own in-house design principles.
+It's intended to serve as an alternative to Material UI for designs that don't adhere to Material Design specifications.
+
+Try Joy UI if you appreciate the comprehensiveness and reliability of Material UI, but don't need all of the additional baggage that comes along with Material Design.
+
+Get started in the [Joy UI docs](/joy-ui/getting-started/overview/).
+
+#### Key features
+
+- **Innovative design:** Free from the constraints of Material Design, Joy UI is where we can innovate and experiment with fresh new ideas in design, UX, and DX.
+- **Flexibility:** Customize with ease, and leverage the power of CSS variables to ensure consistency when making pixel-perfect adjustments.
+
+#### Ideal use cases
+
+- Projects that don't involve Material Design
+- Designs that would benefit from less opinionated defaults when customizing
```

Comment:
```suggestion
- Design systems that would benefit from less opinionated defaults when customizing
```

I would like to add a point about the design system here but not sure if it should be a bullet. My intention is around "If you are building a fresh design system, you should try Joy UI."